Getting started guide (#20)
BitcoinDesign/Guide: guide/getting-started/open-design.md

- Cross-project collaboration
- Project-specific collaboration
Examples in Bitcoin Design
- Guide
- Johns exploring CoinJoins with Wasabi and wider bitcoin audience (cross-project)
- Bosch working with Bitcoin Core GUI (project-specific)

Ideas
- Add quotes and stories from designers

-->

{% include picture.html
   image = "/assets/images/guide/getting-started/open-design/open-design.jpg"
   retina = "/assets/images/guide/getting-started/open-design/open-design@2x.jpg"
   mobile = "/assets/images/guide/getting-started/open-design/open-design-mobile.jpg"
   mobileRetina = "/assets/images/guide/getting-started/open-design/open-design-mobile@2x.jpg"
   alt-text = ""
   caption = ""
   width = 1600
   height = 740
   layout = "full-width"
%}

# Open design

**In a broader view, open design is an alternative method for design and development, based on the free exchange of comprehensive design information.**

Open design tries to balance the independence of individual designers and the collective power of collaboration. The goal is to enable and promote design projects, which are motivated by personal conviction and passion of designers for the greater benefit of a global society.

> In practical terms, open design can manifest itself in a different ways. You might:

Reviewer: ```suggestion
In practical terms, open design can manifest itself in different ways. You might:
```

* help design the interface of your favorite open-source project
* be really interested in a specific topic and collaborate with multiple people and projects in related areas
* create a design resource library (like an icon or illustration set, or a UI Kit) for others to use
* help test and provide feedback to an open-source product
* contribute to this Bitcoin Design Guide (hint, hint)

Whichever it may be, it is a voluntary, public contribution to efforts that you think are worthwhile. From these individual efforts, a network of people and projects emerges that is only loosely bound through personal relationships, communities, or the work itself. This is different from common corporate structures and requires individuals to think and act more independently and be comfortable with ambiguity.

> Open-source development has existed and matured for decades, while open design is not as widely known or practiced. Starting to design for bitcoin products can therefore sometimes be disorienting, as many designers need to adjust not just how they work and collaborate, but also the relationship to their work. No longer is the design an artefact only controlled by just the designer or a small group of people inside an organzation. The design instead becomes part of a public conversation where anyone is invited to help shape it, and group consensus influences the way forward. In open-source software, this public review and debate often leads to better products and services, and this effect ideally carries over to open design.

Reviewer: ```suggestion
Open-source development has existed and matured for decades, while open design is not as widely known or practiced. Starting to design for bitcoin products can therefore sometimes be disorienting, as many designers need to adjust not just how they work and collaborate, but also the relationship to their work. No longer is the design an artefact only controlled by just the designer or a small group of people inside an organization. The design instead becomes part of a public conversation where anyone is invited to help shape it, and group consensus influences the way forward. In open-source software, this public review and debate often lead to better products and services, and this effect ideally carries over to open design.
```

> The best solutions typically come from a comprehensive understanding of the problem to be solved, whether they are technical, human, societal, economic, or otherwise. This makes a multidisciplinary approach essential, with users, designers, developers, translaters, project managers, and others working together.

Reviewer: ```suggestion
The best solutions typically come from a comprehensive understanding of the problem to be solved, whether they are technical, human, societal, economic, or otherwise. This makes a multidisciplinary approach essential, with users, designers, developers, translators, project managers, and others working together.
```

##  Open design and Bitcoin

> Bitcoin has had the spirit of openness deeply embedded from its humble beginnings. The source code is public, participation in the network is public and unrestricted, and transfering value via the Bitcoin protocoal is permissionless as well. Thousands of projects have copied ideas, techniques and code from Bitcoin to experiment and explore different directions. So it provides a fantastic foundation for designers to also contribute in this open and participatory manner.

Reviewer: ```suggestion
Bitcoin has had the spirit of openness deeply embedded from its humble beginnings. The source code is public, participation in the network is public and unrestricted, and transferring value via the Bitcoin protocol is permissionless as well. Thousands of projects have copied ideas, techniques and code from Bitcoin to experiment and explore different directions. So it provides a fantastic foundation for designers to also contribute in this open and participatory manner.
```

Author: Too bad it's wrong. Protocoal sounds like a cool thing :)

One might argue that Bitcoin needs open design in order to stay true to its founding values. The majority of users don't interact directly with a technical protocol, but instead with applications built on top of it. If those applications end up being controlled or manipulated, then it will not matter if the protocol is open. So it is up to the people shaping the open-source application layer to help ensure that the future of Bitcoin remains decentralized and allows for permissionless participation.

## Values

In open design, we... 

- collaborate and don't compete
- are transparent and share our process and work
- make it easy for others to use and build on our ideas and work
- are inclusive and help others
- are open-minded and flexible 

> - work multidisciplinary, because good results require different perspectives and skillsets

Reviewer: ```suggestion
- work multidisciplinary, because good results require different perspectives and skill sets
```

- continuously learn independently, and from each other
- rely on our peer group for free and communal validation of quality
- are accountable for our own actions, and have vested interest in the groups actions and goals

And we value...

- design ethics to make moral and responsible choices
- human design to create products for real needs and desires
- usability to create products that allow users to effectively solve their problems
- accessibility to make products usable by people with the widest possible range of abilities, in the widest possible range of situations
- aesthetics, which have a highly functional element and also influence how people feel and think

## Getting started

> If you are completely new to open design, it can feel tricky to find a starting point in the Bitcoin ecosystem. Not only is the technology complex, but there are also few design resources available. A good starting is the [Contribute]({{ '/contribute' | relative_url }}) page, and here are a few extra tips to help you find your way around in general:

Reviewer: ```suggestion
If you are completely new to open design, it can feel tricky to find a starting point in the Bitcoin ecosystem. Not only is the technology complex, but there are also few design resources available. A good starting point is the [Contribute]({{ '/contribute' | relative_url }}) page, and here are a few extra tips to help you find your way around in general:
```

- Join the [Bitcoin Design Community](http://bitcoindesigners.org) on Slack and see what others are discussing. A good way to get started is by asking questions and joining conversations that interest you. Our [Community Calls](https://github.com/BitcoinDesign/Meta/issues?q=is%3Aissue+is%3Aopen+Community+Call) and [Design Review](https://github.com/BitcoinDesign/Meta/issues?q=is%3Aissue+is%3Aopen+Design+Review+Call+) calls are also great ways to get active.

> - Download a Bitcoin wallet and experiment with small amounts. You can alse use Testnet, which allows you use all the features, but with worthless test bitcoin. Pay attention to the experience and come up with ways to improve it.

Reviewer: ```suggestion
- Download a Bitcoin wallet and experiment with small amounts. You can also use Testnet, which allows you to use all the features, but with worthless test bitcoin. Pay attention to the experience and come up with ways to improve it.
```

- Redesign a Bitcoin application of your choice. Even as a brief exercise, a redesign is a great way to develop deeper understanding of how something works and why certain decisions were made.
- Read, watch and listen. There is a lot of great material out there to learn more about Bitcoin. Design-focused content is unfortunately not as common but expanding into related materials (like finance) might help.

> - Review your tools and workflows, and the work you produce, and consider how you can make it more public and accesssible for others.

Reviewer: ```suggestion
- Review your tools and workflows, and the work you produce, and consider how you can make it more public and accessible for others.
```

## Licensing

Licenses help ensure that bad actors cannot abuse this open system by taking the published work and pretend it's their own, patent it, redistribute it under a commercial license, or otherwise profit from it. A variety of licenses have been created by the open-source movement to accomodate different types of projects and situations.

> This guide is published under three different licenses ([MIT license](LICENSE-MIT), [Apache License 2.0](LICENSE-APACHE), and [Creative Commons Attribution 4.0 International License](LICENSE-CC-BY)). Anyone who wants to remix it, or parts of it, may choose which one to publish their work under. You can read the full license text [here](LICENSE).

Reviewer: Remember to add the links in for these 👍

Author: Good call.

---

> There is enourmous space for creativity. To find a starting point, take a look at the [visual-language of bitcoin]({{ '/guide/getting-started/visual-language/' | relative_url }}).

Reviewer: ```suggestion
There is enormous space for creativity. To find a starting point, take a look at the [visual-language of bitcoin]({{ '/guide/getting-started/visual-language/' | relative_url }}).
```
